docs/source/en/training.mdx

```diff
@@ -63,7 +63,7 @@ If you like, you can create a smaller subset of the full dataset to fine-tune on
 
 <a id='trainer'></a>
 
-## Train
+## Train your PyTorch model with the Trainer
```

Comment:
No this section groups both training with PyTorch and TensorFlow with framework-specific blocks, so leave this as it was.